### ISSUE TITLE
colour coding is not switched off after it was saved to layout and analysis was re-loaded

### ISSUE THREAD

**Issue by JianmeiFan** [datagrok-ai/public#1071]
**Steps to reproduce:**
open analysis
enable colour coding on a column
save layout
re-load analysis
try to switch off the colour coding on the column
**Actual result:** colour coding is not switched off. There is an inconsistency between column (coloured) and colour coding dialog (colouring is shown as 'off').
Issue is reproduced for linear, conditional and categorical colouring.
*Grid Color coding is under Auto*
![image-2021-06-17-14-22-41-851](https://user-images.githubusercontent.com/100243033/192322108-f3bd665d-8987-448f-8ea0-f70df6aafac3.png)

**Comment by onuf** [comment]
@JianmeiFan the color is propagated before/after layout application:
![image](https://user-images.githubusercontent.com/56073962/195136805-3329b7ae-9172-48f7-8ec5-1ca19e4c8193.png)
![image](https://user-images.githubusercontent.com/56073962/195136910-731dad87-29f5-4c02-b41e-cd45cec485ca.png)

**Comment by onuf** [comment]
@katerina-stepanova is it reproduced in native DG, or specifically in the ID app?

**Comment by katerina-stepanova** [comment]
@onuf Yes, it is reproduced in native DG too: apply colours > save layout > open table again > re-apply saved layout.
It applies to all types of colouring (conditional, linear, categorical), '.color-coding-type' tag is missing when saved layout is re-applied.

**Comment by skalkin** [comment]
This is strange, I just tested it on public.datagrok.ai, and everything seems to be in order. @onuf , it might make sense for you to meet with @katerina-stepanova to figure our what's happening

**Comment by katerina-stepanova** [comment]
@skalkin @onuf I've just checked it on public.datagrok.ai, it can be reproduced.
More detailed steps:
1. Apply some colour coding (linear, conditional or categorical)
2. Save layout
3. **Close and re-open the same table, or refresh browser page and open the same table again**
4. Re-apply saved layout
5. Check column's context menu and colour coding dialog

**Comment by onuf** [comment]
Thanks @katerina-stepanova, I managed to reproduce it yesterday, already working on a fix for that, should be ready today

**Comment by onuf** [comment]
Fixed, waiting for deployment & testing on our dev environment

**Comment by onuf** [comment]
Fixed in `1.8.3`
